hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { getByTestId, render, screen } from "testing/utils";
+import { mockHardware, mockPendingOrdersInTable } from "testing/mockData";
+import TeamPendingOrderTable from "components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable";
+import { within, waitFor } from "testing/utils";
+import { fireEvent } from "@testing-library/react";
```

Comment:
pls export fireEvent from testing/utils please and you have 2 imports from testing/utils so merge those together